UploadDaemon_Test/SymbolAnalysis/SymbolCollectionTest.cs

```diff
@@ -5,6 +5,7 @@
 using System.IO;
 using System.Collections.Generic;
 using System.IO.Abstractions;
+using UploadDaemon.Utils;
 
 [TestFixture]
 public class SymbolCollectionTest
```

Comment:
Move to `UploadDaemon.SymbolAnalysis`? Remove corresponding using statement if you decide to do so.